src/auth/auth.controller.ts

```diff
@@ -0,0 +1,35 @@
+import { Controller, Body, Post, HttpCode, UseGuards } from '@nestjs/common';
+import { AuthService } from './auth.service';
+import { LocalAuthGuard } from './guards/local-auth.guard';
+import {
+  SignInRequestDto,
+  SignInResponseDto,
+  SignUpRequestDto,
+  SignUpResponseDto,
```

Comment:
let's experiment a little and drop the `Dto` - just name it `SignInRequest`, `SignInResponse` etc.